pkg/controller/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -334,7 +334,7 @@ func NewNodeLifecycleController(
 		secondaryEvictionLimiterQPS: secondaryEvictionLimiterQPS,
 		largeClusterThreshold:       largeClusterThreshold,
 		unhealthyZoneThreshold:      unhealthyZoneThreshold,
-		nodeUpdateQueue:             workqueue.NewNamed("node_lifecycle_controller"),
+		nodeUpdateQueue:             workqueue.NewWithConfig(workqueue.QueueConfig{Name: "ode_lifecycle_controller"}),
```

Comment:
typo: ode_lifecycle_controller, should be n ode_lifecycle_controller

Reply:
thanks your good catch, will update as soon as possible

Reply:
it is fixed